apps/mobile/src/components/InsideRouter/atoms.ts

```diff
@@ -18,6 +27,16 @@ export const refreshBtcPriceActionAtom = atom<undefined, [], Task<boolean>>(
   (get, set) => {
     const api = get(privateApiAtom)
     const {t} = get(translationAtom)
+    const btcPriceLastUpdateAt = get(btcPriceLastUpdateAtAtom)
+    const timeDifferenceSinceLastUpdate = Math.round(
```

Comment:
This is also a good solution. I would prefer to use miliseconds and `BTC_PRICE_UPDATE_TRIGGER_THRESHOLD_MILISECONDS` instead. 

Is `Math.round` really needed in this case? Let's not complicate the code unnecessarily.